### PR TITLE
ICLR19 - Updates to rebuttal-stage script

### PR DESCRIPTION
**Pull request by mohituniyal**
and corresponding updates in invitations.py

### CODE REVIEW

**Comment by mohituniyal** [comment]
If I am understanding this correctly, @mspector I believe this is already happening at line45 in rebuttal-stage.py ( review.writers = [iclr19.CONFERENCE_ID]), where we are setting writers from ["ICLR.cc/2019/Conference", "ICLR.cc/2019/Conference/Paper<XXXX>/AnonReviewer<Y>"] to just ["ICLR.cc/2019/Conference"].

**Comment by mspector** [comment]
@mohituniyal line 45 is changing the writers of the existing review, yes. what I suggested above is to change the reply field of the official review invitation, such that new reviews created *after* the script is run are not editable by users.

so, something like this (starting on line 33):

```
for review_inv in official_review_invs:
    review_inv.reply['readers']['values'] = ['everyone']
    review_inv.reply['writers']['values'] = ['ICLR.cc/2019/Conference']
    client.post_invitation(review_inv)
    ...
```

**Comment by mohituniyal** [comment]
> @mohituniyal line 45 is changing the writers of the existing review, yes. what I suggested above is to change the reply field of the official review invitation, such that new reviews created _after_ the script is run are not editable by users.
> 
> so, something like this (starting on line 33):
> 
> ```
> for review_inv in official_review_invs:
>     review_inv.reply['readers']['values'] = ['everyone']
>     review_inv.reply['writers']['values'] = ['ICLR.cc/2019/Conference']
>     client.post_invitation(review_inv)
>     ...
> ```
> EDIT: I'm re-reading my comment again and realized that it was not precisely worded. Let me know if this is clearer

Thanks for clearing it. I have made the change.
As you can see, I am now assigning a dict to the writers field. This seems like a more straight forward approach compared to first popping out "values-regex" and then adding "values".

File: venues/ICLR.cc/2019/Conference/python/invitations.py [hunk 1]
```diff
@@ -301,11 +301,12 @@
...
     'Add_Bid': iclr19.add_bid.to_json(),
     'Official_Comment': official_comment_template,
-    'Add_Revision': add_revision_template,
+    'Paper_Revision': add_revision_template,
     'Official_Review': official_review_template,
     'Meta_Review': meta_review_template,
     'Public_Comment': public_comment_template,
-    'Withdraw_Submission': withdraw_submission_template
+    'Withdraw_Submission': withdraw_submission_template,
+    'Review_Revision' : revise_review_template
```
[the remark this answers is not in the document]
Comment (mohituniyal) [reply venues/ICLR.cc/2019/Conference/python/invitations.py]: Yeah, it's just a label that we pass to the rebuttal stage script. On the UI the button will have the text revision for both paper revisions and review revisions.

File: venues/ICLR.cc/2019/Conference/python/rebuttal-stage.py [hunk 2]
```diff
@@ -27,36 +27,50 @@
...
 
-                review_revision_inv = invitations.enable_invitation(
-                    'Revise_Review', target_paper=paper)
-                review_revision_inv.id = review_revision_inv.id.replace('<reviewer_id>', reviewer_id)
-                review_revision_inv.reply['referent'] = review.id
-                review_revision_inv.invitees = review.signatures
+                    review_revision_inv = invitations.enable_invitation(
+                        'Review_Revision', target_paper=review)
+                    
+                    paper_number = review.invitation.split('Paper')[1].split('/')[0]
```
[the remark this answers is not in the document]
Comment (mohituniyal) [reply venues/ICLR.cc/2019/Conference/python/rebuttal-stage.py]: Good point. Thanks.

**Pull request merged**